invenio_rdm_records/services/config.py

```diff
@@ -119,6 +119,11 @@ def archive_download_enabled(record, ctx):
     return current_app.config["RDM_ARCHIVE_DOWNLOAD_ENABLED"]
 
 
+def groups_enabled(record, ctx):
```

Comment:
```suggestion
def _groups_enabled(record, ctx):
```
we could make it internal for this file, otherwise there is a potential to mistake it for the decorator implemented below